### DISCUSSION TITLE
Translation to Portuguese Brazilian (pt-br)

### DISCUSSION THREAD

**Discussion by JeepWillys58** [PeterStrick/ViVeTool-GUI#74] [Ideas]
Hi dear Peter,

Can you open a translation into Brazilian Portuguese?

Let me know, cos I believe I can help with this work.

Best Regards.
JeepWillys58

**Comment by PeterStrick** [comment]
> Hi dear Peter,
> 
> Can you open a translation into Brazilian Portuguese?
> 
> Let me know, cos I believe I can help with this work.
> 
> Best Regards.
> JeepWillys58
> 
> 

Hi, You can use [Weblate](https://weblate.rawrr.dev/projects/vivetool-gui/-/pt_BR/) to translate ViVeTool GUI into Brazilian Portuguese (pt-br)

**Reply by JeepWillys58** [reply]
I tryed to find it before, thanks for the link, I will do it!

Regards!

**Comment by JeepWillys58** [comment]
Hi Peter I follow your link, but it don't let login using my Weblate account, can you help me?

**Reply by PeterStrick** [reply]
> Hi Peter I follow your link, but it don't let login using my Weblate account, can you help me?

Have you created a Weblate Account on the Site?

**Reply by JeepWillys58** [reply]
I had already created an account a few days ago that was associated with my GitHub account, but I couldn't log in using the link you sent me, so I tried to create another one, but I didn't receive the confirmation email either.

Link to my Weblate profile:
https://hosted.weblate.org/user/JeepWillys58/

Maybe I'm doing something wrong?

Thanks in advanced

**Reply by PeterStrick** [reply]
> I had already created an account a few days ago that was associated with my GitHub account, but I couldn't log in using the link you sent me, so I tried to create another one, but I didn't receive the confirmation email either.
> 
> Link to my Weblate profile:
> https://hosted.weblate.org/user/JeepWillys58/
> 
> Maybe I'm doing something wrong?
> 
> Thanks in advanced

weblate.rawrr.dev is using it's own Weblate Accounts

You cannot use a hosted.weblate.org Account on weblate.rawrr.dev

**Reply by JeepWillys58** [reply]
Hi again Peter.

I open your link, click register, add my details and receive a confirmation message that I will receive an email to confirm registration, but since you gave me the link it hasn't arrived, I even tried logging in using GitHub and it simply didn't work. it works.

I tried using both Opera Browser and Firefox.

![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/cacbd759-e0ca-46ba-8b44-41939ec75357)

PS: I used the same data in another project, https://weblate.kuma.pet/, I registered by clicking on the "Create account with GitHub" button and it worked normally, I received the confirmation email and completed the registration and with your link didn't work...

![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/dd3d8578-6b52-410e-a9c9-25428e5908a2)

**Reply by PeterStrick** [reply]
> Hi again Peter.
> 
> I open your link, click register, add my details and receive a confirmation message that I will receive an email to confirm registration, but since you gave me the link it hasn't arrived, I even tried logging in using GitHub and it simply didn't work. it works.
> 
> I tried using both Opera Browser and Firefox.
> 
> ![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/cacbd759-e0ca-46ba-8b44-41939ec75357)
> 
> PS: I used the same data in another project, https://weblate.kuma.pet/, I registered by clicking on the "Create account with GitHub" button and it worked normally, I received the confirmation email and completed the registration and with your link didn't work...
> 
> ![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/dd3d8578-6b52-410e-a9c9-25428e5908a2)
> 

Have you checked that the confirmation Email didn't land up in your Spam Folder?

**Reply by JeepWillys58** [reply]
No Peter, it's not.

**Comment by JeepWillys58** [comment]
When I try to register wirh GitHub, the page just refresh and clean all information added. But don't register.

**Comment by JeepWillys58** [comment]
Look
https://streamable.com/l7zo4e

**Comment by PeterStrick** [comment]
I'll investigate this once I'm at work

**Comment by PeterStrick** [comment]
Sorry, there was a Server Misconfuguration error causing Sign Ups to not go through. It's working now

**Comment by JeepWillys58** [comment]
My dear @PeterStrick, I can only say.... 
Thanks for fixing this! 
So soon we will be working together! 
Gratitude!
Kind Regards
@JeepWillys58

**Comment by JeepWillys58** [comment]
Hello again dear @PeterStrick,

I tried again, then I see your message telling me that it was fixed and it still doesn't work, try it in Opera and Firefox Private Navigation using the credentials I show above, email, username... The system shows the message that a message will be sent confirmation email to the address provided on the registration form, but it does not arrive. I do this for the first time today, 1 hour ago.

Sorry for the inconvenience, but I believe this may be happening to others who are also trying to help with the project.

Wait for your replay.

Yours sincerely
@JeepWillys58

**Reply by PeterStrick** [reply]
> Hello again dear @PeterStrick,
> 
> I tried again, then I see your message telling me that it was fixed and it still doesn't work, try it in Opera and Firefox Private Navigation using the credentials I show above, email, username... The system shows the message that a message will be sent confirmation email to the address provided on the registration form, but it does not arrive. I do this for the first time today, 1 hour ago.
> 
> Sorry for the inconvenience, but I believe this may be happening to others who are also trying to help with the project.
> 
> Wait for your replay.
> 
> Yours sincerely
> @JeepWillys58

There was another error that appeared after upgrading Weblate that is now fixed, I was able to get an Email sent earlier so try again now

**Reply by JeepWillys58** [reply]
Okay I will do it right now.

**Reply by JeepWillys58** [reply]
Hi again @PeterStrick 

Sorry to say, but still not working on my side...

![Captura de tela 2023-10-31 121458](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/6dfeede1-ac18-40e7-ba04-ed61fca03f34)
![Captura de tela 2023-10-31 125803](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/1a329119-a1ad-4d42-b1a0-17f9b95f498f)
![Captura de tela 2023-10-31 145654](https://github.com/PeterStrick/ViVeTool-GUI/assets/34031626/2a466c0d-8f65-4ac8-a06e-53aa514fe2c7)

**Reply by PeterStrick** [reply]
Could you try signing up through GitHub or Google?

**Reply by JeepWillys58** [reply]
Using GitHub works.

**Reply by JeepWillys58** [reply]
Could you close this case, please?

Thanks in advance.

Regards.
@JeepWillys58
